diff --git a/Android.bp b/Android.bp
index 8e11707..a156cb8 100644
--- a/Android.bp
+++ b/Android.bp
@@ -62,8 +62,10 @@ rust_defaults {
 rust_test_host {
     name: "vulkano_host_test_src_lib",
     defaults: ["vulkano_test_defaults"],
+    // Manually disabled as these tests depend on specific graphics libraries
+    // being available on the machine running the tests.
     test_options: {
-        unit_test: true,
+        unit_test: false,
     },
 }
 
