diff --git a/Android.bp b/Android.bp
index 8e11707..a156cb8 100644
--- a/Android.bp
+++ b/Android.bp
@@ -62,9 +62,11 @@ rust_test {
     ],
     test_suites: ["general-tests"],
     auto_gen_config: true,
+    // Manually disabled as these tests depend on specific graphics libraries
+    // being available on the machine running the tests.
     test_options: {
-        unit_test: true,
+        unit_test: false,
     },
     edition: "2018",
     rustlibs: [
 
